plugins/discourse-ai/lib/summarization/entry_point.rb

```diff
@@ -21,6 +21,28 @@ def inject_into(plugin)
           scope.can_see_summary?(object.topic)
         end
 
+        plugin.add_to_serializer(
+          :topic_view,
+          :ai_summary,
+          include_condition: -> do
+            DiscoursePluginRegistry.apply_modifier(:serialize_ai_summary, false) &&
+              scope.can_see_summary?(object.topic)
+          end,
+        ) do
+          summary =
+            object.topic.ai_summaries.find_by(summary_type: AiSummary.summary_types[:complete])
```

Comment:
Will this always get the latest version?

Reply:
There's a [unique constraint in `ai_summary` model](https://github.com/discourse/discourse/blob/0c41ff06809caf5254bd4f87e073ea6e64a81bc5/plugins/discourse-ai/app/models/ai_summary.rb#L71) ensuring only one complete summary per topic exists.